### DISCUSSION TITLE
The title is obscured

### DISCUSSION THREAD

**Discussion by Trung15010802** [imaNNeo/fl_chart#1591] [Q&A]
The start and the and title is obscured? How to fix this? Right now i'm using padding like temp solution
![image](https://github.com/imaNNeo/fl_chart/assets/90593296/8b3db212-94fa-4968-a4d7-80f7ebf7b199)

**Comment by imaNNeo** [comment]
Did you set `clipData`?
Can you provide a reproducible code?
